pkg/metrics/innerring.go

```diff
@@ -2,32 +2,44 @@ package metrics
 
 import "github.com/prometheus/client_golang/prometheus"
 
-const innerRingSubsystem = "object"
+const innerRingNameSpace = "neofs_ir"
+
+// FIXME: drop after v0.38.0 release: #2347.
+const innerRingNameSpaceDeprecated = storageNodeNameSpace
+const innerRingSubsystemDeprecated = objectSubsystem
 
 // InnerRingServiceMetrics contains metrics collected by inner ring.
 type InnerRingServiceMetrics struct {
-	epoch prometheus.Gauge
+	epoch           prometheus.Gauge
+	epochDeprecated prometheus.Gauge
 }
 
 // NewInnerRingMetrics returns new instance of metrics collectors for inner ring.
 func NewInnerRingMetrics() InnerRingServiceMetrics {
-	var (
-		epoch = prometheus.NewGauge(prometheus.GaugeOpts{
-			Namespace: namespace,
-			Subsystem: innerRingSubsystem,
-			Name:      "epoch",
-			Help:      "Current epoch as seen by inner-ring node.",
-		})
-	)
-
+	epoch := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: innerRingNameSpace,
+		Subsystem: stateSubsystem,
+		Name:      "epoch",
+		Help:      "Current epoch as seen by inner-ring node.",
+	})
 	prometheus.MustRegister(epoch)
 
+	epochDeprecated := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: innerRingNameSpaceDeprecated,
+		Subsystem: innerRingSubsystemDeprecated,
+		Name:      "epoch",
+		Help:      "Current epoch as seen by inner-ring node.",
```

Comment:
Mark as deprecated in the help?

Reply:
sure (the line is too long intentionally if you dont mind)